temporalio/worker/workflow_sandbox/_restrictions.py

```diff
@@ -1032,7 +1065,7 @@ def __getitem__(self, key: Any) -> Any:
     )
     __str__ = _RestrictedProxyLookup(str)  # type: ignore
     __bytes__ = _RestrictedProxyLookup(bytes)
-    __format__ = _RestrictedProxyLookup()  # type: ignore
+    __format__ = _RestrictedProxyLookup(format)  # type: ignore
```

Comment:
This fixes a small bug in the existing sandboxing. Without this, `pathlib.Path` fields are broken.